migrate_asset_relation_to_master_item_reference.sql

```diff
@@ -0,0 +1,55 @@
+-- This script will migrate from an asset relation into a master item reference field.
+-- This will remove all existing values for the field first, so if you already have values in the field, they will be lost.
+
+-- Set these variables
+declare @targetMasterItemReferenceFieldItemGuid uniqueidentifier = '1ce1558f-e4d7-490f-87d3-47dbd4ebdfd3';
+declare @sourceAssetRelationTypeId int = 2;
```

Comment:
```suggestion
declare @targetMasterItemReferenceFieldItemGuid uniqueidentifier = '';
declare @sourceAssetRelationTypeId int = 0;
```

Just to ensure that we don't hit something unintended 😃